sklearn/dummy.py

```diff
@@ -435,7 +435,7 @@ def fit(self, X, y, sample_weight=None):
 
             self.constant = check_array(self.constant,
                                         accept_sparse=['csr', 'csc', 'coo'],
-                                        ensure_2d=False)
+                                        ensure_2d=False, ensure_min_samples=0)
```

Comment:
The problem is that `self.constant` is often a scalar. If I don not disable the `ensure_min_samples` check it raises a `TypeError` because the number of samples is undefined in that case (see `_num_samples`).